controllers/_current.json

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
@osama891 Whtat is the purpose of this file?

Reply:
explained in the auto-migrations thread